spec/function.dd

```diff
@@ -1117,27 +1117,26 @@ $(H2 $(LNAME2 function-overloading, Function Overloading))
         $(LI exact match)
         )
 
-        $(P Each argument (including any $(CODE this) pointer) is
-        compared against the function's corresponding parameter, to
+        $(P Each argument (including any $(CODE this) reference) is
+        compared against the function's corresponding parameter to
         determine the match level for that argument. The match level
         for a function is the $(I worst) match level of each of its
         arguments.)
 
         $(P Literals do not match $(CODE ref) or $(CODE out) parameters.)
 
-
         $(P If two or more functions have the same match level,
         then $(LNAME2 partial-ordering, $(I partial ordering))
-        is used to try to find the best match.
+        is used to disambiguate to find the best match.
         Partial ordering finds the most specialized function.
         If neither function is more specialized than the other,
         then it is an ambiguity error.
-        Partial ordering is determined for functions $(CODE f())
-        and $(CODE g()) by taking the parameter types of $(CODE f()),
+        Partial ordering is determined for functions $(I f)
+        and $(I g) by taking the parameter types of $(I f),
         constructing a list of arguments by taking the default values
-        of those types, and attempting to match them against $(CODE g()).
-        If it succeeds, then $(CODE g()) is at least as specialized
-        as $(CODE f()).
+        of those types, and attempting to match them against $(I g).
+        If it succeeds, then $(I g) is at least as specialized
+        as $(I f).
```

Comment:
I think we should use code blocks throughout for the function names `f` and `g`
```suggestion
        as `f`.
```

Reply:
Since the functions in the example were not f and g, I decided to use the more abstract mathematical notation.